ApplicationLibCode/Application/RiaPreferences.cpp

```diff
@@ -855,6 +855,15 @@ QString RiaPreferences::gridCalculationExpressionFolder() const
     return m_gridCalculationExpressionFolder().path();
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RiaPreferences::summaryCalculationExpressionFolder() const
+{
+    // TODO: use separate or same folder?
+    return m_gridCalculationExpressionFolder().path();
```

Comment:
Use separate folder for summary expressions

Reply:
Fixed!